frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/vm-wrapper.ts

```diff
@@ -33,6 +34,14 @@ import { VirtualMachineImportModel, VirtualMachineModel } from '../../../models'
 import { buildOwnerReferenceForModel } from '../../../utils';
 import { transformDevices } from '../../../selectors/vm/devices';
 
+export const getDataVolumeTemplateSpec = (dv: V1alpha1DataVolume): V1DataVolumeTemplateSpec =>
```

Comment:
what about 
```suggestion
export const convertToDataVolumeTemplateSpec = (dv: V1alpha1DataVolume): V1DataVolumeTemplateSpec =>
```

or something similar to this?

Reply:
also it probably shouldn't reside in vm-wrapper